### PR TITLE
Create Affects Config

### PR DESCRIPTION
**Pull request by ArlonAntonius**
Speaks for itself

### CODE REVIEW

File: src/Affects/Config/composer.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        "psr-4": {
+            "Tenancy\\Affects\\Config\\": ""
+        }
+    },
+    "authors": [
+        {
+            "name": "Daniël Klabbers",
+            "email": "daniel@klabbers.email",
+            "homepage": "http://luceos.com"
+        }
```
Comment (luceos) [inline src/Affects/Config/composer.json:24]: @ArlonAntonius someone is missing here
^

**Review by luceos — CHANGES_REQUESTED** [review]

**Comment by ArlonAntonius** [comment]
Let's first create tests before merging.

File: src/Affects/Config/composer.json (hunk 1 above)
Comment (ArlonAntonius) [reply src/Affects/Config/composer.json:24]: Thanks :D

**Comment by luceos** [comment]
🎉 nice job!

**Pull request merged**